module/os/linux/spl/spl-condvar.c

```diff
@@ -198,6 +198,13 @@ __cv_wait_sig(kcondvar_t *cvp, kmutex_t *mp)
 }
 EXPORT_SYMBOL(__cv_wait_sig);
 
+void
+__cv_wait_idle(kcondvar_t *cvp, kmutex_t *mp)
+{
+	cv_wait_common(cvp, mp, TASK_UNINTERRUPTIBLE, 0);
```

Comment:
This issue with this on Linux is that when sleeping as `TASK_UNINTERRUPTIBLE` the process counts towards the systems load average.  This is why we hijacked `cv_wait_sig()` to so we could sleep as `TASK_INTERRUPTIBLE`.

Reply:
OK. It doesn't matter on FreeBSD. I'll make it INTERRUPTIBLE